Sources/Runner/Commands/Runner.swift

```diff
@@ -78,23 +77,23 @@ func runDanger(logger: Logger) throws {
                            "\(importExternalDeps.joined(separator: ", ")),",
                            "this might take some time.")
 
-            try Folder(path: ".").createFileIfNeeded(withName: "_dangerfile_imports.swift")
-            let tempDangerfile = try File(path: "_dangerfile_imports.swift")
-            try tempDangerfile.write(string: importExternalDeps.joined(separator: "\n"))
-            defer { try? tempDangerfile.delete() }
+            let tempDangerfile = "_dangerfile_imports.swift"
+            try importExternalDeps.joined(separator: "\n").write(toFile: tempDangerfile, atomically: false, encoding: .utf8)
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Runner/Commands/Runner.swift;
  Line: 81;
-->

- :warning: Line should be 120 characters or less: currently 124 characters (`line_length`)